deploy/helm/grafana-operator/templates/rbac.yaml

```diff
@@ -204,11 +204,6 @@ rules:
       - patch
       - update
       - watch
-  - apiGroups:
```

Comment:
So we want OCP users to be able to use helm.
It would be nice to add a variable like `isOpenShift` that could be set to false by default.

Reply:
that's done now, can you please have another look?